src/main/external-resources/renderers/FetchTV.conf

```diff
@@ -0,0 +1,39 @@
+#----------------------------------------------------------------------------
+# Profile for Fetch TV (Mighty and Mini) @ https://www.fetchtv.com.au/
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Fetch TV
+RendererIcon = fetch-tv.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
+# ============================================================================
+#
+
+UserAgentSearch = Linux/3.14.28-1.6.*DLNADOC/1.50
```

Comment:
@Sami32 thanks for the feedback. Renderer does not seem to uniquely identify itself so I am not sure what to change it to. Any suggestions?